### DISCUSSION TITLE
Ordem de serviço com segunda via

### DISCUSSION THREAD

**Discussion by Marcelo197002** [RamonSilva20/mapos#1961] [Ideas]
Olá, pessoal senti a falta de uma segunda via para a ordem de serviço, pois da maneira que está o gasto de papel será grande.
Minha sugestão se possível colocar as informações duplicadas em uma mesma folha A4.
Agradecido.

**Comment by radsonpatrick** [comment]
Opa Marcelo tudo bom ? Me chama aí que faço pra você!! 98981725021
Lembrando NAO VOU TE COBRAR !

**Reply by SuperPorte** [reply]
Poxa.... podia disponibilizar na principal essa opção hein....
Eu configurei aqui pra imprimir em A5 e estou cortando as folhas todas as vezes.
Se der pra fazer uma especie de canhoto, seria melhor ainda.

**Reply by evertonterra** [reply]
> Opa Marcelo tudo bom ? Me chama aí que faço pra você!! 98981725021 Lembrando NAO VOU TE COBRAR !

Olá @radsonpatrick te chamei no WhatsApp se puder me ajudar lhe agradeço

**Comment by radsonpatrick** [comment]
Boa noite pessoal acabei de abrir um PR com essa nova feature segue a PR 

#1971
